src/components/Signup.vue

```diff
@@ -48,11 +48,30 @@
             {{ item }}
           </el-tag>
         </el-form-item>
+        <el-form-item
```

Comment:
Please note 2 things:
1. For the `pilot` we don't need Signup IP input at all.
2. We need those __cool__ inputs in our main (i.e. `develop`) branch.